homeworks/andrey.kushch_andrey241991/8_components_templates/components-templates/src/app/image-gallery/image-gallery.component.html

```diff
@@ -0,0 +1,11 @@
+<section class="container">
+  <ng-container *ngFor="let image of images; index as i">
+    <div class="item" [ngClass]="i === imgPosition ? 'item--active' : ''">
```

Comment:
```suggestion
    <div class="item" [сlass.item--active]="i === imgPosition">
```
Для одного динамічного класу краще використовуй такий синтаксис. Він більш простий у читанні